modules/navigation.js

```diff
@@ -0,0 +1,20 @@
+
+const sectionOfBooks = document.querySelector('#books-section');
+const sectionContact = document.querySelector('#section-contact');
+const sectionForm = document.querySelector('#form-section');
+
+export default function selection(id) {
```

Comment:
- [ ] Great function implementation, but as the project required, kindly change that to an arrow function so it can follow the ES6 best practices.

Reply:
This change has been done well